saleor/payment/gateways/adyen/webhooks.py

```diff
@@ -709,7 +729,7 @@ def get_or_create_adyen_partial_payments(
     already_existing_partial_payments = Payment.objects.filter(
         psp_reference__in=[p.psp_reference for p in new_payments],
         partial=True,
-        checkout_id=payment.checkout_id,
+        checkout_id=payment.checkout_id,  # type: ignore
```

Comment:
Why ignore the type?

Reply:
Getting error:  
```
Incompatible type for lookup 'checkout_id': (got "Optional[UUID]", expected "Union[UUID, str]")
```
Don't know why as I didn't touch this part of the code. It looks like the assumption for typing is wrong here as the passed checkout_id can be None. I didn't want to spend time debugging this as I wanted to create PR asap.